ci/glci/azure.py

```diff
@@ -89,6 +91,17 @@ def get_image_url(self, image_name: str):
         return f"https://{self.sa_name}.blob.core.windows.net/{self.container_name}/{image_name}?{container_sas}"
 
 
+class Azmp_operation_state(Enum):
```

Comment:
nit: according to PEP8, use CamelCase for class names (snake_case for variables..)